Expecto.Tests/Tests.fs

```diff
@@ -644,33 +644,33 @@ let expecto =
       ]
 
       testList "list is empty" [
-        testCase "pass" <| fun _ ->
+        testCase "list pass" <| fun _ ->
```

Comment:
Why change these? It's about **globally** unique names. I don't mind the change though, but I'm curious.

Reply:
`pass` -> `list pass` because there are checks  for an array and list and they have the same name I would split them to different test lists.
![image](https://user-images.githubusercontent.com/7689103/30658033-bf8ec670-9e39-11e7-96fc-2e623c8c932c.png)